diff --git a/franka_description/robots/panda.transmission.xacro b/franka_description/robots/panda.transmission.xacro
index 343bd7a..17c15bb 100644
--- a/franka_description/robots/panda.transmission.xacro
+++ b/franka_description/robots/panda.transmission.xacro
@@ -7,6 +7,7 @@
         <gazebo>
             <plugin name="gazebo_ros_control" filename="libgazebo_ros_control.so">
                 <!-- <robotNamespace>/${robot_name}</robotNamespace> -->
+                <legacyModeNS>true</legacyModeNS>
             </plugin>
         </gazebo>
 
